### PR TITLE
Pass full ciphertext byte slice to checking func rather than subset

### LINKED ISSUE #40: checkCipherTextLength() being called on wrong target byte slice
..it's being called with the chunk of encrypted bytes that's already been pulled out of the full ciphertext (which is the purpose of the PlainText() func):

https://github.com/ovotech/mantle/blob/8a507d829d669c53e8ec6b447847f14bf9047d27/crypt/decrypt.go#L50:L51
```
	plaintext, err := PlainText(x.Filepath)
	checkCipherTextLength(plaintext)
```

this is causing the panic: 'CipherText was shorter than the smallest possible generated CipherText' when the chunk of encrypted bytes happens to be shorter than 124 bytes (completely valid if the plaintext happens to be small).

checkCipherTextLength() needs to be called on the full ciphertext, instead.

### PR DESCRIPTION
**Pull request by eversC**
Variablize dekLength and encDekLength

Add more helpful text to the ciphertext length panic

fixes #40

### CODE REVIEW

**Comment by eversC** [comment]
I tested this with some ciphertexts created with current mantle/master, including one which ,after splitting out into cipher + dek + nonce, had a cipher < 124 chars (so was previously failing).

File: crypt/crypt.go [hunk 1]
```diff
@@ -44,6 +44,8 @@ var defaultOptions = Defaults{}
 var Parser = flags.NewParser(&defaultOptions, flags.Default)
 
 var nonceLength = 12
```
Comment (JackTreble) [inline crypt/crypt.go]: are these expected to change length at all?
I also can't do multi line suggestions so ill do them one by one
```suggestion
const nonceLength = 12
```

File: crypt/crypt.go [hunk 2]
```diff
@@ -44,6 +44,8 @@ var defaultOptions = Defaults{}
...
+var dekLength = 32
```
Comment (JackTreble) [inline crypt/crypt.go]:
```suggestion
const dekLength = 32
```

File: crypt/crypt.go [hunk 3]
```diff
@@ -44,6 +44,8 @@ var defaultOptions = Defaults{}
...
+var encDekLength = 113
```
Comment (JackTreble) [inline crypt/crypt.go]:
```suggestion
const encDekLength = 113
```

Comment (eversC) [reply crypt/crypt.go]: fixed in a3388f9

File: crypt/crypt.go (hunk 2 above)
Comment (eversC) [reply crypt/crypt.go]: fixed in a3388f9

File: crypt/crypt.go (hunk 1 above)
Comment (eversC) [reply crypt/crypt.go]: fixed in a3388f9

**Pull request merged**